tests/Responses/ajax/product/index_response.json

```diff
@@ -25,25 +25,15 @@
                 "code": "MUG_BB",
                 "options": [],
                 "averageRating": 0,
-                "images": [],
-                "_links": {
-                    "self": {
-                        "href": "\/api\/v1\/products\/MUG_BB"
```

Comment:
Isn't it a BC? Do we have any possibility to sustain it? 

On the other hand, this filed should not be hire in the first place

Reply:
Without Admin API Bundle it's impossible to generate those URLs. If you have Admin API Bundle installed, it will have those links included.

It feels like it should be mentioned in the upgrade instructions though. Most likely no one uses those AJAX responses in any other way than for autovompletes.